src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -105,32 +120,32 @@ void CascadiaSettings::SaveAll() const
 // - <none>
 // Return Value:
 // - a JsonObject which is an equivalent serialization of this object.
-JsonObject CascadiaSettings::ToJson() const
+Json::Value CascadiaSettings::ToJson() const
 {
-    // _globals.ToJson will initialize the settings object will all the global
-    // settings in the root of the object.
-    winrt::Windows::Data::Json::JsonObject jsonObject = _globals.ToJson();
+    Json::Value root;
+    // TODO: Globals
 
-    JsonArray schemesArray{};
-    const auto& colorSchemes = _globals.GetColorSchemes();
-    for (auto& scheme : colorSchemes)
+    // TODO: put the keybindings in the globals (now that the order doesn't
+    // really matter).
+
+    Json::Value profilesArray;
+    for (const auto& profile : _profiles)
     {
-        schemesArray.Append(scheme.ToJson());
+        profilesArray.append(profile.ToJson());
     }
 
-    JsonArray profilesArray{};
-    for (auto& profile : _profiles)
+    Json::Value schemesArray;
+    const auto& colorSchemes = _globals.GetColorSchemes();
+    for (auto& scheme : colorSchemes)
     {
-        profilesArray.Append(profile.ToJson());
+        schemesArray.append(scheme.ToJson());
     }
 
-    jsonObject.Insert(PROFILES_KEY, profilesArray);
-    jsonObject.Insert(SCHEMES_KEY, schemesArray);
-
-    jsonObject.Insert(KEYBINDINGS_KEY,
-                      _globals.GetKeybindings().ToJson());
+    root[GLOBALS_KEY.data()] = _globals.ToJson();
+    root[PROFILES_KEY.data()] = profilesArray;
+    root[SCHEMES_KEY.data()] = schemesArray;

```

Comment:
per the conversation in chat; should this be `static_cast<std::string>(SCHEMES_KEY)`?

Reply:
(because `.data()` invokes the `const T*` `strlen/wcslen` constructor, whereas this invokes the `string_view` size+knownLength constructor)

Reply:
I forgot where we landed on that